data/api/client-server/definitions/event_filter.yaml

```diff
@@ -14,7 +14,7 @@
 title: EventFilter
 properties:
   limit:
-    description: The maximum number of events to return.
+    description: The maximum number of events to return. If not specified defaults to 10.
```

Comment:
I'm not entirely sure we should mandate this, or that we *can* mandate it, without going through the MSC process: it may well be that non-synapse implementations use a different default.

I think what we need to say instead is that if it's not specified, servers should use an implementation-specific default.

The `limit` param on [`GET /threads`](https://spec.matrix.org/v1.6/client-server-api/#get_matrixclientv1roomsroomidthreads) has sensible wording here.